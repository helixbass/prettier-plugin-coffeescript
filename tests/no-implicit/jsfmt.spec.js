run_spec(__dirname, ['coffeescript'], {noImplicit: []})
run_spec(__dirname, ['coffeescript'], {noImplicit: ['objectBraces']})
run_spec(__dirname, ['coffeescript'], {noImplicit: ['callParens']})
run_spec(__dirname, ['coffeescript'], {
  noImplicit: ['objectBraces', 'callParens'],
})
