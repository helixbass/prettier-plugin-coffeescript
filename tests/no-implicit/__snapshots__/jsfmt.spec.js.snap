// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`no-implicit.coffee 1`] = `
a: b c
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
a: b c

`;

exports[`no-implicit.coffee 2`] = `
a: b c
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
{ a: b c }

`;

exports[`no-implicit.coffee 3`] = `
a: b c
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
a: b(c)

`;

exports[`no-implicit.coffee 4`] = `
a: b c
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
{ a: b(c) }

`;
